File: packages/java/src/main/java/com/tesote/sdk/errors/SyncInProgressException.java
package com.tesote.sdk.errors;

/**
 * 409 — a sync session for the bank connection is already running.
 * The active session id is in {@link #responseBody()} as
 * {@code current_session_id}.
 */
public final class SyncInProgressException extends ApiException {
    public SyncInProgressException(String message, String errorCode, int httpStatus,
                                   String requestId, String errorId, Integer retryAfter,
                                   String responseBody, RequestSummary requestSummary,
                                   int attempts, Throwable cause) {
        super(message, errorCode, httpStatus, requestId, errorId,
                retryAfter, responseBody, requestSummary, attempts, cause);
    }
}

File: packages/csharp/src/Errors/SyncInProgressException.cs
using System;

namespace Tesote.Sdk.Errors;

/// <summary>409 SYNC_IN_PROGRESS — another sync session is currently active.</summary>
public sealed class SyncInProgressException : ApiException
{
    /// <summary>Construct with the full required-field set.</summary>
    public SyncInProgressException(
        string? message, string? errorCode, int httpStatus,
        string? requestId, string? errorId, int? retryAfter,
        string? responseBody, RequestSummary? requestSummary,
        int attempts, Exception? cause)
        : base(message, errorCode, httpStatus, requestId, errorId,
               retryAfter, responseBody, requestSummary, attempts, cause)
    {
    }
}
